packages/web/.storybook/.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "../../../.babelrc"
```

Comment:
I thought it will be loaded as default

Reply:
unfortunately, not 😅

Reply:
> By default, Storybook loads your root .babelrc file and load those configurations. 
https://storybook.js.org/configurations/custom-babel-config/

But it only reads the **"root"** of `web/` workspace, if the babelrc is out of its scope, it wont be loaded.

Reply:
Shhh.. I thought it is duplicated